browser/src/canvas/sections/CommentSection.ts

```diff
@@ -120,7 +120,7 @@ export class Comment extends CanvasSectionObject {
 		this.sectionProperties.showSelectedCoordinate = true; // Writer.
 
 		if (app.map._docLayer._docType === 'presentation' || app.map._docLayer._docType === 'drawing') {
-			this.sectionProperties.parthash = this.sectionProperties.data.parthash;
+			this.sectionProperties.parthash = parseInt(this.sectionProperties.data.parthash);
```

Comment:
if `parthash` inside `data` should be always a number, shouldn't we do it once when we receive that data and assign to `sectionProperties`? so we don't have to convert again.
Can we have `number` type assigned to it so TS compiler will warn about incorrect usage?

Reply:
I think this is the right place. This is constructor and sectionProperties.data is an input parameter.